layout/_third-party/analytics/busuanzi-counter.swig

```diff
@@ -3,18 +3,25 @@
   <script async src="https://busuanzi.ibruce.info/busuanzi/2.3/busuanzi.pure.mini.js"></script>
 
   {% if theme.busuanzi_count.total_visitors %}
-    <span class="site-uv" title="{{ __('footer.total_visitors') }}">
+    <span class="post-meta-item-icon">
       <i class="fa fa-{{ theme.busuanzi_count.total_visitors_icon }}"></i>
+    </span>
+    <span class="site-uv" title="{{ __('footer.total_visitors') }}">
       <span class="busuanzi-value" id="busuanzi_value_site_uv"></span>
     </span>
   {% endif %}
 
+  {% if theme.busuanzi_count.total_visitors and theme.busuanzi_count.total_views %}
```

Comment:
![image](https://user-images.githubusercontent.com/16944225/50641886-ab36f880-0f69-11e9-8dea-e117447204bf.png)

Mb not need divider here? Because it's one plugin.

Reply:
Add an option for users to decide then ?

Reply:
The separator `|` is displayed in NexT v5.x

Reply:
I mean about internal `|` separator. 

This is pretty:
![image](https://user-images.githubusercontent.com/16944225/50643794-526a5e80-0f6f-11e9-8514-920ddfdb033f.png)

This is not so nice:
![image](https://user-images.githubusercontent.com/16944225/50641886-ab36f880-0f69-11e9-8dea-e117447204bf.png)

Reply:
But, if you want this separator, need to add style like this:

![image](https://user-images.githubusercontent.com/16944225/50644099-2ac7c600-0f70-11e9-8633-ca7091dd18b3.png)

Reply:
Oh, it's added, I forget to update the screenshot.
![2019-01-03 11 00 03](https://user-images.githubusercontent.com/16272760/50644300-659b1f80-0fab-11e9-9f68-fb9f27e5cb0b.png)
It looks fine